components/serializers/keymap.hpp

```diff
@@ -0,0 +1,59 @@
+
```

Comment:
I think this file best demonstrates everything I don't like about OSG serialization, but what can I do?...